source/iostreams.tex

```diff
@@ -626,6 +626,7 @@
 \synopsis{Header \tcode{<ios>} synopsis}
```

Comment:
Eew. This synopsis should *replace* the rSec2 heading just above, but the stable name is a bit silly then.